src/Protocol/Protocol_1_8.cpp

```diff
@@ -2576,7 +2576,7 @@ void cProtocol_1_8_0::HandlePacketPlayerPosLook(cByteBuffer & a_ByteBuffer)
 
 void cProtocol_1_8_0::HandlePacketPluginMessage(cByteBuffer & a_ByteBuffer)
 {
-	// https://wiki.vg/index.php?title=Plugin_channels&oldid=14089#MC.7CAdvCmd
+	// https://wiki.vg/index.php?title=Plugin_channels&oldid=14089#MC.7CAdvCmd -- page probably no longer exists
```

Comment:
https://minecraft.wiki/w/Minecraft_Wiki:Projects/wiki.vg_merge/Plugin_channels?oldid=2768100 matched via web archive + looking up date of modify 2 July 2018 in the new wiki edit history.

Reply:
I couldn't find this one on the new wiki

Reply:
yeah if you look for it with the old oldid, it'll show you another page